ui-tests/src/test/java/org/openmrs/reference/page/ProviderPage.java

```diff
@@ -17,7 +17,7 @@
     private static final By IDENTIFIER_FIELD = By.cssSelector("#content input[name=\"identifier\"]");
     private static final By PERSON_FIELD = By.id("person_id_selection");
     private static final By SAVE = By.cssSelector("#content input[type=\"submit\"]");
```

Comment:
you can change all the rest to make this page readable else no need of this single one